src/helpers/youtubeFetcher.js

```diff
@@ -0,0 +1,30 @@
+const fetch = require('node-fetch');
+const { YOUTUBE_API_KEY, YOUTUBE_CHANNEL_ID } = require('../config');
+
+function transformYoutubeResponseData(data) {
+  return data.items.map((item) => {
+    const { title, publishedAt, description } = item.snippet;
```

Comment:
This destructuring can be done in the argument in place of `item`.

Reply:
Don't forget to destruct `item.snippet.thumbnails` too

Reply:
I've already changed this line once before for destructuring's sake. Let's worry about bigger picture tasks and then we can do sweeps of ideal syntax changes later.

Reply:
See comment above. Lets get this right the first time so that we don't have to go back and fix things.

Reply:
Then suggest a specific change and I'll make it

Reply:
This should do it -
```js
function transformYoutubeResponseData(data) {
  return data.items.map(({ snippet: { id: { videoID, title, publishedAt, description, thumbnails: { high } } }) => {
    return {
      type: 'video',
      name: title,
      videoID: videoId,
      date: publishedAt,
      url: `https://www.youtube.com/watch?v=${videoId}`,
      thumbnail: high,
      title,
      description,
    };
  });
}
```

Reply:
I apologize, I misread your original comment. I see what you mean, but I think it's a bit harder to read when it's destructured in the argument. I think it's right as it is and because it follows a standard across its sibling files. If you think a change of those standards should be made (I could agree with that), but it is out of scope of this pull request.

Reply:
You can alternatively do it like this but we should destructure it all the way regardless - 
```js
function transformYoutubeResponseData({ items }) {
  return items.map((item) => {
    const { snippet: { id: { videoID }, title, publishedAt, description, thumbnails: { high } } } = item;
    return {
      type: 'video',
      name: title,
      videoID: videoId,
      date: publishedAt,
      url: `https://www.youtube.com/watch?v=${videoId}`,
      thumbnail: high,
      title,
      description,
    };
  });
}
```